Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -1170,7 +1168,7 @@ void getSupportedTypes(HashSet<String>& types) const final
         RetainPtr<id> observer = m_sizeChangeObservers.takeFirst();
         [m_synchronizer removeTimeObserver:observer.get()];
     }
-    m_sizeChangeObserverWeakPtrFactory.revokeAll();
+    m_flushCount++;
```

Comment:
Why are we doing this change in this patch, is it needed?
Add some logs to the commit message.

Reply:
Originally, I had made [MediaPlayerPrivateMediaSourceAVFObjC.mm] use a ThreadSafeAndThreadSafeWeakPtr and so having a WeakPtrFactory was no longer needed ; but in any case this was really over the top for what it was used for (just the ability of cancelling the observer once we flushed). so I kept it

Reply:
I've removed it for now